branches.go

```diff
@@ -176,8 +176,9 @@ func (s *BranchesService) UnprotectBranch(pid interface{}, branch string, option
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/branches.html#create-repository-branch
 type CreateBranchOptions struct {
-	Branch *string `url:"branch,omitempty" json:"branch,omitempty"`
-	Ref    *string `url:"ref,omitempty" json:"ref,omitempty"`
+	IssueIid *int    `url:"issue_iid,omitempty" json:"issue_iid,omitempty"`
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms:
```suggestion
	IssueIID *int    `url:"issue_iid,omitempty" json:"issue_iid,omitempty"`
```

Reply:
sorry, I made a mistake, it's a invalid merge.    

I use gitlab-web find this IssueIid by F12, but I missing tests the new struct cannot link issue